index.html

```diff
@@ -97,86 +103,129 @@ <h1>bam2.iobio</h1>
+              </div>
+            </iobio-info-button>
+
+            <iobio-percent-box percent-key="singletons" total-key="total_reads"></iobio-percent-box>
           </iobio-panel>
 
           <iobio-panel>
-            <iobio-percent-box
-              label="Both Mates Mapped"
-              percent-key="both_mates_mapped"
-              total-key="total_reads">
-            </iobio-percent-box>
+            <iobio-info-button label="Both Mates Mapped">
+              <div slot="content">
+                <p>This is custom content for the Both Mates Mapped percent box.</p>
+              </div>
+            </iobio-info-button>
+
+            <iobio-percent-box percent-key="both_mates_mapped" total-key="total_reads"></iobio-percent-box>
           </iobio-panel>
          
           <iobio-panel>
-            <iobio-percent-box
-              label="Duplicates"
-              percent-key="duplicates"
-              total-key="total_reads">
-            </iobio-percent-box>
+            <iobio-info-button label="Duplicates">
+              <div slot="content">
+                <p>This is custom content for the Duplicates percent box.</p>
+              </div>
+            </iobio-info-button>
+
+            <iobio-percent-box percent-key="duplicates" total-key="total_reads"></iobio-percent-box>
           </iobio-panel>
 
         </div>
 
         <div class='histogram-column'>
           <iobio-panel>
-            <iobio-histogram
-              label="Read Coverage Distribution"
-              broker-key="coverage_hist">
-            </iobio-histogram>
+            <iobio-info-button label="Read Coverage Distribution">
+              <div slot="content">
+                <p>This is custom content for the Read Coverage Distribution histogram.</p>
+              </div>
+            </iobio-info-button>
+
+            <iobio-histogram broker-key="coverage_hist"></iobio-histogram>
           </iobio-panel>
 
           <iobio-panel>
-            <iobio-tabs label-1="Fragment Length" label-2="Read Length">
-              <iobio-histogram
-                broker-key="frag_hist"
-                ignore-outliers>
-              </iobio-histogram>
-
-              <iobio-histogram
-                broker-key="length_hist"
-                ignore-outliers>
-              </iobio-histogram>
+            <iobio-tabs>
+              <iobio-tab slot="tab">
+                <iobio-info-button label="Fragment Length" icon-position="left">
+                  <div slot="content">
+                    <p>This is custom content for the Fragment Length histogram.</p>
+                  </div>
+                </iobio-info-button>
+              </iobio-tab>
+              <iobio-tab-panel slot="panel">
+                <iobio-histogram broker-key="frag_hist" ignore-outliers></iobio-histogram>
+              </iobio-tab-panel>
+
+       
+              <iobio-tab slot="tab">
+                <iobio-info-button label="Read Length" icon-position="right">
```

Comment:
Can you make `icon-position="right"` the default so it doesn't need to be set here?